.travis.yml

```diff
@@ -14,10 +14,13 @@ rust:
 os:
 - linux
 before_script:
+- export PATH="$PATH:$HOME/.cargo/bin"
+- which rustfmt || cargo install rustfmt
```

Comment:
If you change this to `cargo +nightly install rustfmt-nightly` this PR should pass as that crate is the one that is updated from now on! Should have remembered that earlier.

May need to do `rustup install nightly` as well if it does not work on stable/beta.

https://crates.io/crates/rustfmt-nightly